dev/AppNotifications/AppNotificationManager.h

```diff
@@ -37,6 +39,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
             [[maybe_unused]] NOTIFICATION_USER_INPUT_DATA const* data,
             [[maybe_unused]] ULONG dataCount) noexcept;
 
+        static winrt::Windows::Foundation::IInspectable Deserialize(winrt::Windows::Foundation::Uri const& uri);
```

Comment:
Why static?

Reply:
Other Deserialize functions like LaunchActivatedEventArgs, FileActivatedEvent args use the static keyword with Deserialize. The ExtensionMap in ExtensionContract.h is expecting a winrt::Windows::Foundation::IInspectable(*factory)(winrt::Windows::Foundation::Uri const& uri). I guess by using the & operator on a static function you get the factory. Compile errors occur when removing the static tag.